### PR TITLE
Update InteractWithGraph.tsx - Fix failures caused by break change of stripping off the async suffix from SK functions

### PR DESCRIPTION
**Pull request by freistli**
Fix failures caused by break change of stripping off the async suffix from SK functions

### Motivation and Context

The auth-api sample doesn't work anymore because of the break change listed in https://github.com/freistli/semantic-kernel/blob/main/docs/decisions/0003-support-multiple-native-function-args.md

Updated name selection heuristic to strip off an "Async" suffix for async methods. 

### Description

KernelServer service reports similar errors the sample is calling:

 Result: Function 'InvokeFunction', Invocation id '6dcbc5af-9484-4357-a5ac-c91b585dfac5': An exception was thrown by the invocation.
Exception: System.AggregateException: One or more errors occurred. (Function not available: Function not available emailskill.getmyemailaddressasync)

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

**Review by teresaqhoang — APPROVED** [review]

**Comment by shawncal** [comment]
@freistli We've fixed the failing precheck (Python Integration Tests) on our end, but we cannot update your branch or queue a new build. It looks like you have not yet checked the "Allow edits from maintainers" box that would normally show this on the righth column:
image

Could you please update (merge) the latest changes from main? And, if you see the option, check the "allow edits from maintainers" box in this PR? That will allow us to merge directly into main upon successful checks, or apply further merges if needed. Thanks!

**Comment by freistli** [comment]
Thanks @shawncal , just merged the main branch in the pull request. I cannot see "allow edits from maintainers" option on my right column. Now the PR status is waiting for OCTO group reviewers' approval.

**Review by shawncal — APPROVED** [review]

**Comment by shawncal** [comment]
Awesome, queued for merge. Thanks @freistli for the fix!

**Pull request merged**
